### ISSUE TITLE
JavaScript parsing is overly aggressive

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#1]
```
What steps will reproduce the problem?
1. Note the yellow warnings on many files and projects
2.

What is the expected output? What do you see instead?

I would expect Preferences -> JavaScript -> Validator -> Errors/Warnings to be 
able to provide more control.

Interestingly, even though Optional Semi-colon is only visible if you Enable 
semantic validation, you can enable it, change Optional Semi-colon to ignore 
and then Disable semantic.  


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 5 Jan 2011 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Created https://bugs.eclipse.org/bugs/show_bug.cgi?id=335265
```

Original comment by `paul.beu...@gmail.com` on 24 Jan 2011 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
1.2 will make the default options less harsh and add a JSLint option
```

Original comment by `paul.beu...@gmail.com` on 2 Feb 2011 at 5:54
- Changed state: **Started**
- Added labels: **Milestone-Release1.2.0**

**Comment by GoogleCodeExporter** [comment]
```
Semicolon default addressed with 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=64

Open additional issues for more parsing improvements
```

Original comment by `paul.beu...@gmail.com` on 20 Feb 2011 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
